pandas/core/groupby/generic.py

```diff
@@ -450,7 +450,9 @@ def transform(self, func, *args, **kwargs):
 
             result = concat(results).sort_index()
         else:
-            result = Series()
+            with warnings.catch_warnings():
```

Comment:
don’t catch warnings simply return Series(dtype=object)

Reply:
Isn't this running the risk of breaking things downstream because the return type of this method might change? 

I was rather thinking about passing `dtype=float64`, but that runs the risk of not being removed when the default dtype change is actually implemented and might lead to inconsistencies in future releases.

This is why I went for silencing the warning for now: No change right now & easier to discover/remove in the future.

Reply:
i would rather not in the code
it’s easier to see what night change in tests